src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback, useMemo } from 'react';
+import { useEffect, useState } from 'react';
+import { cn, pressable, text as themeText } from '../../styles/theme';
+import { version } from '../../version';
+import { useIcon } from '../hooks/useIcon';
+import type { WalletDropdownFundLinkReact } from '../types';
+import { getWindowDimensions } from '../utils/getWindowDimensions';
+
+export function WalletDropdownFundLink({
```

Comment:
nit. alphabetical order

Reply:
fixed in new diff, updated in `types.mdx` as well